.github/workflows/cml.yml

```diff
@@ -3,87 +3,221 @@ name: benchmarks
 on:
   workflow_dispatch:
```

Comment:
What triggers do we want to use? Right now it's just manual trigger.

Reply:
I'd start with a schedule [at 00:01 on Sunday](https://crontab.guru/#1_0_*_*_0): `"1 0 * * 0"`.